.travis.yml

```diff
@@ -3,20 +3,33 @@ language: php
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPENDENCIES=""
+        - XDEBUG_MODE="coverage"
         - TEST_COVERAGE=true
-    - php: 7.1
+    - php: 7.3
+      env:
+          - DEPENDENCIES=""
+          - PHP_CS=true
```

Comment:
this is not used anymore?

Reply:
It is ( https://travis-ci.org/github/prooph/common/jobs/752820990 ) , CS is only executed once though now, not on every PHP Version.

https://xdebug.org/docs/upgrade_guide for reference about XDEBUG_MODE in xdebug 3